@here/harp-datasource-protocol/lib/WorkerServiceProtocol.ts

```diff
@@ -87,6 +90,13 @@ export namespace WorkerServiceProtocol {
         targetServiceId: string;
     }
 
+    /**
+     * Test if `error` thrown by [[CreateServiceRequest]] was caused by unknown type of service.
+     */
+    export function isUnknownServiceError(error: Error): boolean {
```

Comment:
Shouldn't a similar error be thrown as a ``warn`` when trying to destroy a service?

Reply:
Maybe yes, maybe not, this patch is about improving UX of our public API w.r.t connecting to decoders. 
So from one side is out of scope.
From other side, destroying unknown service is IMO, super internal error, which doesn't occur frequently whereas "not registered" `omv-tiler` or `omv-decoder` is quite probable, beginner level issue that's why it has dedicated error and logging.

Reply:
You're right, connecting is a way more common use-case and will affect more the UX, but think that warning on destruction is relevant. 

You're right it might be out of scope though,  so we have to make sure to tackle it in the future.